### PR TITLE
Feature/event for off chain transactions

### PR DESCRIPTION
**Pull request by tcrosado**
Implemented a function to inform through events offchain transactions executed earlier.
This transaction receives a wallet address list and the respective Transaction's Merkle tree for each wallet as a list.

### CODE REVIEW

File: test/appCoinsIAB.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ contract('AppCoinsIAB', function(accounts) {
 		expect(appSFinalBalance).to.be.equal(appSInitBalance+(price*appSShare));
```
Comment (DioPires) [inline test/appCoinsIAB.js:59]: #### Tests:
```diff 
+ 40 passing
```
#### Test coverage: 
83.43 %

Comment (DioPires) [inline test/appCoinsIAB.js:59]: #### Tests:
```diff 
+ 40 passing
```
#### Test coverage: 
83.43 %

**Review by DioPires — APPROVED** [review]

File: test/appCoinsIAB.js (hunk 1 above)
Comment (DioPires) [inline test/appCoinsIAB.js:59]: #### Tests:
```diff 
+ 40 passing
```
#### Test coverage: 
83.43 %

Comment (DioPires) [inline test/appCoinsIAB.js:59]: #### Tests:
```diff 
+ 40 passing
```
#### Test coverage: 
83.43 %

**Review by GustavoRSSilva — APPROVED** [review]

**Pull request merged**